pkg/smi/health_test.go

```diff
@@ -0,0 +1,48 @@
+package smi
+
+import (
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestCheckSMICrdsExist(t *testing.T) {
+	testCases := []struct {
+		testName         string
+		reqVersions      map[string]string
+		candidateVersion []string
+		expErr           bool
+	}{
+		{
+			testName:         "default",
+			reqVersions:      map[string]string{"hi": "bye"},
+			candidateVersion: []string{"bye"},
+			expErr:           true,
```

Comment:
I would rename this to something like `expected` or `expOk` because I read this as "I expect this test case to return an error."